youtube_dl/extractor/neteasemusic.py

```diff
@@ -94,14 +95,19 @@ def _call_player_api(self, song_id, bitrate):
         url = 'https://interface3.music.163.com/eapi/song/enhance/player/url'
         data, headers = self.make_player_api_request_data_and_headers(song_id, bitrate)
         try:
-            return self._download_json(
+            msg = 'empty result, maybe you should use a proper proxy server located in China'
```

Comment:
We have a method that tells people this, that can be called later.
```suggestion
            msg = 'empty result'
```